### PR TITLE
COMMERCE-6125 JSONUtil changes

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan unit tests are failing :(

**Comment by alecsloan** [comment]
ci:test:relevant

File: portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java [hunk 1]
```diff
@@ -220,15 +223,18 @@ public void testIsArray() {
...
 
 	@Test
-	public void testisValid() {
+	public void testIsJSONObject() {
 		Assert.assertTrue(
 			"null is an empty JSON string", JSONUtil.isJSONObject(null));
 
-		Assert.assertTrue("\"\" is an empty JSON string", JSONUtil.isJSONObject(""));
+		Assert.assertTrue(
+			"\"\" is an empty JSON string", JSONUtil.isJSONObject(""));
```
Comment (dsitu) [inline portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java:231]: @alecsloan we should use `StringPool.BLANK` here.
```suggestion
			"\"\" is an empty JSON string", JSONUtil.isJSONObject(StringPool.BLANK));
```

File: portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java [hunk 2]
```diff
@@ -197,18 +197,18 @@ public void testHasValue() {
...
-
-		Assert.assertFalse(
-			"\"\" is not a JSON array", JSONUtil.isJSONArray(""));
+		Assert.assertTrue(
+			"null is an empty JSON array", JSONUtil.isJSONArray(null));
 
-		Assert.assertFalse(
-			"{} is not a JSON array", JSONUtil.isJSONArray("{}"));
+		Assert.assertTrue(
+			"\"\" is an empty JSON array", JSONUtil.isJSONArray(""));
```
Comment (dsitu) [inline portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java:204]: @alecsloan we should use `StringPool.BLANK` here.
```suggestion
			"\"\" is an empty JSON array", JSONUtil.isJSONArray(StingPool.BLANK));
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan can you please re-base and push this PR?

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan it seems you need to run baseline

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan again baseline issues :(

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan again issues :(

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**